File: python/0069-Sqrt(x).py
    def mySqrt(self, x):
        """
        :type x: int
        :rtype: int
        """
        i = 0
        j = 46341
        
        while i + 1 < j:
            m = int((i+j)/2)
            if x < m * m:
                j = m
            else:
                i = m
        
        return i

File: CPP/0001-0100/0069-Sqrt(x).cpp
class Solution {
public:
    int mySqrt(int x) {
        int i(0), j(46341);

        while (j - i > 1) {
            int m = (i + j) / 2;
            x < m * m ? j = m : i = m;
        }

        return i;
    }
};
